chapter02/worksheets/solution2.1.7.1_AR.scala

```diff
@@ -0,0 +1,25 @@
+/** author: Andreas Röhler */
+
+/** 
+Exercise 2.1.7.1
+Find all pairs i, j within (0, 1, ..., 9) such that i + 4 ∗ j > i ∗ j.
+Hint: use flatMap and filter.
+  */
+
+val a = (0 to 9).sliding(2).toList
```

Comment:
Here this seems to be taking pairs like (0, 1), (1, 2), (2, 3) and so on. But the question was about all possible pairs of i, j. For example, (3, 8) or (5, 1) and so on.

Reply:
Checked in a fix.